templates/Dockerfile-cli.template

```diff
@@ -31,7 +31,9 @@ ENTRYPOINT [ "/entrypoint.sh" ]
 %%/NEW_PHP%%
```

Comment:
The `NEW_PHP` tag can be removed from this Dockerfile template, since all PHP versions will make use of the bits inside it.